packaged/lisk_snapshot.sh

```diff
@@ -35,9 +35,9 @@ SNAPSHOT_CONFIG="$PWD/etc/snapshot.json"
 TARGET_DB_NAME="$( jq -r .db.database "$SNAPSHOT_CONFIG" )"
 LOG_LOCATION="$( jq -r .logFileName "$SNAPSHOT_CONFIG" )"
 
-LISK_CONFIG="config.json"
+LISK_CUSTOM_CONFIG=$( get_lisk_custom_config "$(pwd)/etc/pm2-lisk.json" )
```

Comment:
Where does this `get_lisk_custom_config` function come from?

Reply:
it comes from shared.sh

Reply:
I don't think this needed to be changed: we want the core custom config file (not the snapshot's) since it's used to find the core database name.

Reply:
That should still be the case, it's looking in pm2-lisk.json and not pm2-snapshot.json. That said, we could also do `LISK_CUSTOM_CONFIG="config.json"`, though it'd be a bit inconsistent with how we get that elsewhere

Reply:
Fair enough, let's stay consistent.